<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SyvDcExistingNodeDropAreaWidget</class>
 <widget class="QWidget" name="C_SyvDcExistingNodeDropAreaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>314</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeGbxTransparent" name="pc_GroupBoxValid">
     <property name="title">
      <string>Valid</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="pc_LabelIcon">
        <property name="text">
         <string>Icon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="stw_opensyde_gui_elements::C_OgeLabPropertiesSubHeading" name="pc_LabelSerialNumber">
        <property name="text">
         <string>SN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="stw_opensyde_gui_elements::C_OgePubTabClose" name="pc_PushButtonClose">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeGbxTransparent" name="pc_GroupBoxEmpty">
     <property name="title">
      <string>Empty</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>263</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="stw_opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelContent">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>262</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeGbxTransparent</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbxTransparent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGroupItem</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupItem.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabPropertiesSubHeading</class>
   <extends>QLabel</extends>
   <header>C_OgeLabPropertiesSubHeading.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubTabClose</class>
   <extends>QPushButton</extends>
   <header>C_OgePubTabClose.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
